src/index.ts

```diff
@@ -3,8 +3,10 @@ import { AnalyticsReporter } from './infra/AnalyticsReporter';
 import { ConversionTrackingReporter } from './infra/ConversionTrackingReporter';
 import { SearchAnalyticsReporter } from './infra/SearchAnalyticsReporter';
 import { PagesAnalyticsReporter } from './infra/PagesAnalyticsReporter';
+import ChatAnalyticsReporter from './infra/ChatAnalyticsReporter';
 import { AnalyticsConfig, PagesAnalyticsConfig, SearchAnalyticsConfig } from './models';
 import { AnalyticsService, SearchAnalyticsService, PagesAnalyticsService, ConversionTrackingService } from './services';
+import ChatAnalyticsConfig from './models/config/ChatAnalyticsConfig';
```

Comment:
can you switch to a non-default import for these please?

Reply:
Yep!